src/components/shapes/attributes.js

```diff
@@ -110,10 +110,13 @@ module.exports = templatedArray('shape', {
 
     layer: {
         valType: 'enumerated',
-        values: ['below', 'above'],
+        values: ['below', 'above', 'between'],
```

Comment:
@LiamConnors @emilykl 
Anything else we may call this option?

Reply:
I agree this one isn't quite descriptive enough, but having a hard time thinking of a better one.

I think it's important that the name indicates where it is relative to the other shape layer, which is tough because `below` and `above` are also not very descriptive.


Brainstorming some alternatives...

- `above-gridlines` -- not clear to user how this differs from `above` (and unclear whether above or below traces)
- `below-traces` -- not clear to user how this differs from `below` (and unclear whether above or below gridlines)
- `between-gridlines-and-traces` -- probably too verbose (but very clear)

Reply:
Not that I have a strong opinion on this (any name is fine for me), but another option would be to introduce "above-traces" and "below-gridlines" as synonyms for "above" and "below", then "between" becomes more obvious.